databases/week2/homework2-part3.sql

```diff
@@ -0,0 +1,27 @@
+-- Active: 1719747438964@@127.0.0.1@3306@test
+-- Get all the tasks assigned to users whose email ends in @spotify.com
+
+SELECT * FROM task
```

Comment:
The queries are correct, but I think you need to use other tables and values according to the description of the homework